pkg/tests/upgrade_tests/shapshot.go

```diff
@@ -215,7 +216,21 @@ func expectedSchemas(extstate testhelpers.TestOptions) []string {
 	return considerSchemas
 }
 
+func alterSearchPath(t *testing.T, db *pgxpool.Pool) {
```

Comment:
if this function is to be called `alterSearchPath`, then it should accept a parameter saying the new path, otherwise the function name should reflect its just alerting search path to only `pg_temp`.